lightning: refine dsecription of check-requirements (#14239)
pingcap/docs: tidb-lightning/tidb-lightning-configuration.md

### tidb-lightning task configuration

[lightning]

> # Checks whether the cluster satisfies the minimum requirement before starting, and check whether TiKV has more than 10% free space left after startup.

Author: ```suggestion
# Checks whether the cluster satisfies the minimum requirement before starting the task,, and check whether TiKV has more than 10% free space left after startup.
```

> # Checks whether the cluster satisfies the minimum requirement before starting the task, and check whether TiKV has more than 10% free space left after startup.

Author: ```suggestion
# Checks whether the cluster satisfies the minimum requirement before starting the task, and check whether TiKV has more than 10% free space left during running time.
```

| --enable-checkpoint *bool* | Whether to enable checkpoints (default = true) | `checkpoint.enable` |
| --analyze *level* | Analyze tables after importing. Available values are "required", "optional" (default value), and "off" | `post-restore.analyze` |
| --checksum *level* | Compare checksum after importing. Available values are "required" (default value), "optional", and "off" | `post-restore.checksum` |

> | --check-requirements *bool* | Check cluster version compatibility before starting the task, and check whether TiKV has more than 10% free space left after startup. (default = true) | `lightning.check-requirements` |

Author: ```suggestion
| --check-requirements *bool* | Check cluster version compatibility before starting the task, and check whether TiKV has more than 10% free space left during running time. (default = true) | `lightning.check-requirements` |
```
